src/plugin.js

```diff
@@ -312,6 +312,25 @@
             onAction: startPlugin
         });
 
+        const setupMenuItem = editor => {
```

Comment:
No need to surround the code in function. Add them directly like the `addButton`.